protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx

```diff
@@ -79,8 +79,24 @@ function AnalyticsStateDistribution({ geography, selectedGeographicFeature }) {
     </>
   );
 
+  function CommunityCharacteristicsButton({
+    countyName,
+    geography,
+    selectedGeographicFeature,
+  }) {
+    return <CommunityCharacteristics />;
+  }
+
```

Comment:
I think this can be dropped and 92-99 as you have now everything in MainChart 👍

Reply:
Oh right, that was right in front of my face. Thank you!